templates/typescript_samples/samples/generated/$version/$service.$method.js.njk

```diff
@@ -0,0 +1,80 @@
+{% import "../../../../typescript_gapic/_util.njk"  as util %}
```

Comment:
@alexander-fenster, I can't seem to run baseline tests here because the generator fails at this import step.

The error says:

```
Error: template not found: /private/var/tmp/_bazel_sofialeon/58d36e9a44f791c802dfa8f145c3330e/execroot/gapic_generator_typescript/bazel-out/darwin-fastbuild/bin/gapic_generator_typescript.sh.runfiles/gapic_generator_typescript/templates/templates/typescript_gapic/_util.njk
```

Even though I can see the file right there in that path. I've tried recompiling and then running, I'm not sure what to do.

Reply:
The way I've been testing is by copying this file in the same directory as the template, and then importing wtih:

```{% import "./_util.njk"  as util -%}```

Reply:
Not sure if it is typo, missing one dot before `./_util.njk`,  `{% import "../_util.njk" as util -%}`

Reply:
Synced offiline. Need to add config for another path.